project/profiles/capacity4more/modules/c4m/content/c4m_content_share/c4m_content_share.module

```diff
@@ -192,6 +192,32 @@ function c4m_content_share_node_view($node, $view_mode, $langcode) {
 }
 
 /**
+ * Access callback checks if the current user is group member.
+ *
+ * @param string $group_type
+ *   The type of the group.
+ * @param int $gid
+ *   The ID of the group.
+ *
+ * @return void|bool
+ *   TRUE if the current user is group member, FALSE otherwise.
+ */
+function c4m_content_share_access($group_type = NULL, $gid = NULL) {
```

Comment:
@soniCaH 
since `access arguments` are not defined on the _hook_menu()_ then we can probably drop the arguments on this callback, unless we should provide it.